packages/repository/README.md

```diff
@@ -21,213 +21,6 @@ $ npm install --save @loopback/repository
 
 ## Basic use
```

Comment:
Can we change it to `Basic Use`?

Reply:
In loopback.io, we did sentence-case for titles and I thought we'd be sticking to that in the READMEs as well. Is it different for readmes?

Reply:
I was looking at the READMEs for @loopback/core and @loopback/rest and they seem to use the title capitalization rules.  But i also see that the READMEs are not all consistent.  Let me check with @bschrammIBM

Reply:
won't be a blocker for this PR.

Reply:
@shimks , according to @bschrammIBM, you're right!